<ui version="4.0" >
 <class>UICLIToolSettings</class>
 <widget class="QWidget" name="UICLIToolSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Commands Editor</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>3</number>
   </property>
   <property name="leftMargin" >
    <number>5</number>
   </property>
   <property name="topMargin" >
    <number>5</number>
   </property>
   <property name="rightMargin" >
    <number>5</number>
   </property>
   <property name="bottomMargin" >
    <number>5</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="horizontalSpacing" >
      <number>3</number>
     </property>
     <property name="verticalSpacing" >
      <number>3</number>
     </property>
     <item row="0" column="0" colspan="2" >
      <widget class="QLabel" name="lCommands" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Commands :</string>
       </property>
      </widget>
     </item>
     <item rowspan="5" row="1" column="0" >
      <widget class="QListWidget" name="lwCommands" />
     </item>
     <item row="1" column="1" >
      <widget class="QPushButton" name="pbCommandAdd" >
       <property name="text" >
        <string>Add</string>
       </property>
       <property name="icon" >
        <iconset resource="../../resources/resources.qrc" >:/tools/icons/tools/add.png</iconset>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QPushButton" name="pbCommandRemove" >
       <property name="text" >
        <string>Remove</string>
       </property>
       <property name="icon" >
        <iconset resource="../../resources/resources.qrc" >:/tools/icons/tools/remove.png</iconset>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1" >
      <widget class="QPushButton" name="pbCommandDown" >
       <property name="text" >
        <string>Down</string>
       </property>
       <property name="icon" >
        <iconset resource="../../resources/resources.qrc" >:/tools/icons/tools/down.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QPushButton" name="pbCommandUp" >
       <property name="text" >
        <string>Up</string>
       </property>
       <property name="icon" >
        <iconset resource="../../resources/resources.qrc" >:/tools/icons/tools/up.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbCommand" >
     <property name="title" >
      <string>Command</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>8</number>
      </property>
      <property name="leftMargin" >
       <number>5</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>5</number>
      </property>
      <property name="bottomMargin" >
       <number>5</number>
      </property>
      <item>
       <layout class="QGridLayout" >
        <property name="horizontalSpacing" >
         <number>3</number>
        </property>
        <property name="verticalSpacing" >
         <number>3</number>
        </property>
        <item row="0" column="0" colspan="2" >
         <widget class="QLabel" name="lCommandText" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Text :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2" >
         <widget class="QLineEdit" name="leCommandText" />
        </item>
        <item row="2" column="0" colspan="2" >
         <widget class="QLabel" name="lCommandCommand" >
          <property name="text" >
           <string>Command :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLineEdit" name="leCommandCommand" />
        </item>
        <item row="3" column="1" >
         <widget class="QToolButton" name="tbCommandCommand" >
          <property name="text" >
           <string>...</string>
          </property>
          <property name="icon" >
           <iconset resource="../../resources/resources.qrc" >:/tools/icons/tools/file.png</iconset>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2" >
         <widget class="QLabel" name="lCommandArguments" >
          <property name="text" >
           <string>Arguments :</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2" >
         <widget class="QLineEdit" name="leCommandArguments" />
        </item>
        <item row="6" column="0" colspan="2" >
         <widget class="QLabel" name="lCommandWorkingDirectory" >
          <property name="text" >
           <string>Working Directory :</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" >
         <widget class="QLineEdit" name="leCommandWorkingDirectory" />
        </item>
        <item row="7" column="1" >
         <widget class="QToolButton" name="tbCommandWorkingDirectory" >
          <property name="text" >
           <string>...</string>
          </property>
          <property name="icon" >
           <iconset resource="../../resources/resources.qrc" >:/tools/icons/tools/icon.png</iconset>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="2" >
         <widget class="QCheckBox" name="cbCommandSkipOnError" >
          <property name="text" >
           <string>Skip On Error</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="lCommand" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="lCommandParsers" >
          <property name="text" >
           <string>Parsers :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="lwCommandParsers" >
          <property name="maximumSize" >
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbCommandTryAll" >
          <property name="text" >
           <string>Try All</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dbbButtons" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Help|QDialogButtonBox::NoButton|QDialogButtonBox::Reset|QDialogButtonBox::RestoreDefaults|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc" />
 </resources>
 <connections/>
</ui>
